config/routes.rb

```diff
@@ -4,4 +4,6 @@
   get '/auth/facebook', as: :login
   get '/auth/:provider/callback', to: 'sessions#create'
   get '/logout', as: :logout, to: 'sessions#destroy'
+
+  get 'place', to: 'static#place'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.